src/components/Navigation/Navigation.styles.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import mediaQuery from '../../utils/mediaQuery';
+
+export const NavigationWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: var(--smallest-space), var(--small-space);
+  background: var(--white);
+
+  ${mediaQuery.tablet} {
+    padding: 20px;
```

Comment:
It can be converted into variable `var(--small-space)`

Reply:
Done!